db/migrate/20170905210626_create_configuration_templates.rb

```diff
@@ -0,0 +1,12 @@
+class CreateConfigurationTemplates < ActiveRecord::Migration[5.0]
```

Comment:
@bdunne Can you review here as well.  I'm ok with configuration_templates, but could that get confusing with configuration manager based classes?

Reply:
I agree that it will be confusing, but I don't have any better names for it right now.